src/api-doc/api-doc.yaml

```diff
@@ -4436,92 +4438,256 @@ paths:
           example: 1
       requestBody:
         content:
-          application.json:
+          application/json:
             schema:
               $schema: http://json-schema.org/draft-04/schema#
               type: object
               properties:
                 comment:
-                  type: string
-                  description: The actual content of the comment
-                context:
-                  type: string
-                  description: Context of the comment (e.g., "page" or any tag)
-                page:
-                  type: string
-                  description: Page number associated with the comment.(e.g. "project_details")
-                status:
-                  type: string
-                  description: Status of the comment (e.g., "OPEN" / "DRAFT" , "RESOLVED").
-                parent_id:
-                  type: string
-                  description: ID of the parent comment, if applicable
-
+                  oneOf:
+                    - type: object
+                      properties:
+                        id:
+                          type: integer
+                          description: ID of the comment
+                        resource_id:
+                          type: integer
+                          description: ID of the resource associated with the comment
+                        comment:
+                          type: string
+                          description: The actual content of the comment
+                        user_id:
+                          type: string
+                          description: ID of the user who made the comment
+                        parent_id:
+                          type: integer
+                          description: ID of the parent comment, if applicable
+                        status:
+                          type: string
+                          description: Status of the comment (e.g., "OPEN", "DRAFT", "RESOLVED")
+                        resolved_by:
+                          type: string
+                          description: ID of the user who resolved the comment, if applicable
+                        resolved_at:
```

Comment:
"@adithyadinesh0412 resolved_at, resolved_by, created_at, updated_at, deleted_at, commenter, and resolver are not needed in the request body. The response structure should be defined under the 200 status code."